roadmap/implementors-guide/src/types/candidate.md

```diff
@@ -0,0 +1,152 @@
+# Candidate Types
+
+Para candidates are some of the most common types, both within the runtime and on the Node-side.
+
+In a way, this entire guide is about these candidates: how they are scheduled, constructed, backed, included, and challenged.
+
+This section will describe the base candidate type, its components, and abridged counterpart.
+
+## CandidateReceipt
+
+This is the base receipt type. The `GlobalValidationSchedule` and the `LocalValidationData` are technically redundant with the `inner.relay_parent`, which uniquely describes the a block in the blockchain from whose state these values are derived. The [`AbridgedCandidateReceipt`](#abridgedcandidatereceipt) variant is often used instead for this reason.
+
+However, the full CandidateReceipt type is useful as a means of avoiding the implicit dependency on availability of old blockchain state. In situations such as availability and approval, having the full description of the candidate within a self-contained struct is convenient.
+
```

Comment:
Is this only about convenience or are there actual situations where the history is unavailable?

Reply:
Most full nodes, likely including validators, will prune state older than finality.